cmd/router/main.go

```diff
@@ -31,18 +34,19 @@ import (
 	"syscall"
 	"time"
 
-	"github.com/kserve/kserve/pkg/constants"
 	"github.com/pkg/errors"
-
+	flag "github.com/spf13/pflag"
 	"github.com/tidwall/gjson"
+	authnv1 "k8s.io/api/authentication/v1"
+	authzv1 "k8s.io/api/authorization/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
-	"crypto/rand"
-	"math/big"
-
 	"github.com/kserve/kserve/pkg/apis/serving/v1alpha1"
-	flag "github.com/spf13/pflag"
+	"github.com/kserve/kserve/pkg/constants"
```

Comment:
shouldn't we import odh/kserve here instead kserve/kserve?

Reply:
I've have had this thought previously.

But... this is still a local import because this is the name we have in go.mod: https://github.com/opendatahub-io/kserve/blob/4e3f485f8cf1719be85962590085bfd1c8f7a8fc/go.mod#L1

We can change the name of our package, but then we need to rename all imports. I'm not sure it is worth it.